pkgs/games/natural-pong/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, udev, alsa-lib, vulkan-loader, xorg, libxkbcommon, wayland, pkg-config, makeWrapper, rustPlatform, fetchFromGitHub }:
+let
+  buildInputs = [
+    udev alsa-lib vulkan-loader
+    xorg.libX11 xorg.libXcursor xorg.libXrandr xorg.libXi
```

Comment:
Don't use `xorg.`

Reply:
The binary loads the xorg libraries during startup. That's why I had to include the libraries. Is there another solutions? The dependencies come from this document: https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md

Reply:
When I said "do not use xorg.", I  was talking about xorg DOT.

Instead of `xorg.libX11 xorg.libXcursor xorg.libXrandr xorg.libXi`, use `libX11 libXcursor libXrandr libXi`.